cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -433,3 +433,13 @@ endif()
 # complain about unused CMake variable.
 if (CMAKE_CROSSCOMPILING_EMULATOR)
 endif()
+
+# TODO: CMake appends <sysroot>/usr/include to implicit includes; switching to use usr/include will make this redundant.
+file(TO_CMAKE_PATH "${EMSCRIPTEN_SYSROOT}/include" _em_sysroot_include)
```

Comment:
Why not just `set(_em_sysroot_include "${EMSCRIPTEN_SYSROOT}/include")`.. I don't see TO_CMAKE_PATH used in the code system libs such as `Modules/Platform/UnixPaths.cmake`

Reply:
I want to know, too, but I don't have a Windows machine to reproduce it (taken from first CI run):
```
-- Configuring incomplete, errors occurred!
See also "C:/Users/circleci/AppData/Local/Temp/tmpro4jytao/emscripten_test_other_q2r21txp/CMakeFiles/CMakeOutput.log".
emcmake: error: 'cmake -GNinja C:\Users\circleci\project\tests\cmake\cmake_with_emval -DCMAKE_TOOLCHAIN_FILE=C:\Users\circleci\project\cmake\Modules\Platform\Emscripten.cmake -DCMAKE_CROSSCOMPILING_EMULATOR=C:/Users/circleci/emsdk/node/14.18.2_64bit/bin/node.exe;--experimental-wasm-threads' failed (returned 1)
None
None
test_cmake_with_embind_cpp11_mode (test_other.other) ... FAIL
with old browser + --closure=1
configure: cmake -DSET_CUSTOM_SUFFIX_IN_PROJECT=0 C:\Users\circleci\project\tests\cmake\static_lib -DCMAKE_TOOLCHAIN_FILE=C:\Users\circleci\project\cmake\Modules\Platform\Emscripten.cmake -DCMAKE_CROSSCOMPILING_EMULATOR=C:/Users/circleci/emsdk/node/14.18.2_64bit/bin/node.exe;--experimental-wasm-threads -G Ninja
CMake Error at C:/Users/circleci/AppData/Local/Temp/tmpv730s3t1/emscripten_test_other_q3dtlspf/CMakeFiles/3.23.2/CMakeCCompiler.cmake:69 (set):
  Syntax error in cmake code at

    C:/Users/circleci/AppData/Local/Temp/tmpv730s3t1/emscripten_test_other_q3dtlspf/CMakeFiles/3.23.2/CMakeCCompiler.cmake:69

  when parsing string

    C:\Users\circleci/cache/sysroot/include

  Invalid escape sequence \U
Call Stack (most recent call first):
  CMakeLists.txt:3 (project)
```

Reply:
Oh I see, in that case make its the value of EMSCRIPTEN_SYSROOT (from _emcache_output) that needs to have TO_CMAKE_PATH run on it.

I'm OK with this change landed as is can investigate as a followup.

Reply:
Yes it looks like we should be doing `file(TO_CMAKE_PATH _emcache_output _emcache_output)` above on line 230

Reply:
Yeah, sounds reasonable and can be easily understood as sanitising the input from external command.